src/azure-cli/azure/cli/command_modules/containerapp/_constants.py

```diff
@@ -74,7 +74,7 @@
   - push: ["$Registry/{{image_name}}"]
     timeout: 1800
 """
-DEFAULT_PORT = 8080  # used for no dockerfile scenario; not the hello world image
+DEFAULT_PORT = 0  # used for no dockerfile scenario; not the hello world image. Auto detect the port
```

Comment:
May I ask if this change would bring a breaking change to users?

Reply:
It shouldn't be, we'll detect the port for them.